src/imap_mag/imapProcessing.py

```diff
@@ -34,10 +35,44 @@ class HKProcessor(FileProcessor):
     xtcePacketDefinition: Path
 
     def initialize(self, config: appConfig.AppConfig) -> None:
-        self.xtcePacketDefinition = config.packet_definition.hk
+        # first try the file path as is, then in the same directory as the module, then fallback to a default
+        pythonModuleRelativePath = Path(
```

Comment:
This is something we might need to do in more places than just here. Could be a utility?

Reply:
Agreed - when we have a use for it I will do the refactor!